pkg/ddevapp/hostname_mgt.go

```diff
@@ -110,13 +111,16 @@ func (app *DdevApp) AddHostsEntriesIfNeeded() error {
 			util.Warning("Unable to open hosts file: %v", err)
 			continue
 		}
-		util.Warning("The hostname %s is not currently resolvable, trying to add it to the hosts file", name)
-
-		out, err := escalateToAddHostEntry(name, dockerIP)
-		if err != nil {
-			return err
+		if !govalidator.IsDNSName(name) {
+			util.Warning("DDEV cannot add unresolvable hostnames like `%s` to your hosts file.\nSee docs for more info, https://ddev.readthedocs.io/en/stable/users/configuration/config/#additional_hostnames.", name)
```

Comment:
The anchor doesn't work work with dot.

```suggestion
			util.Warning("DDEV cannot add unresolvable hostnames like `%s` to your hosts file.\nSee docs for more info, https://ddev.readthedocs.io/en/stable/users/configuration/config/#additional_hostnames", name)
```

Reply:
Thanks, wonder where that came from.